src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -149,78 +150,6 @@ def get_default_paa_trust_store(root_path: pathlib.Path) -> pathlib.Path:
         return pathlib.Path.cwd()
 
 
-def type_matches(received_value, desired_type):
```

Comment:
Please set all of these removed methods as direct aliases to chip.testing.utilities

Reply:
As long as there is no `__all__ = ...` in the file, the `type_matches` will be visible here because it's imported as `from chip.testing.utilities import cluster_id_str, id_str, type_matches`.

Reply:
I think an announcement would be fine. I'd like to remove the aliases before we officially create a package, but there's a limited set of people that will be affected.